res/skins/Deere/tool_bar.xml

```diff
@@ -310,33 +310,7 @@
             <BackPath>image/mixxx_logo.svg</BackPath>
           </WidgetGroup>
 
-          <WidgetGroup>
```

Comment:
I think we should keep the skin Icon for the menu. 2.1 users will miss it.

Reply:
ok